crates/re_edit_ui/src/material.rs

```diff
@@ -0,0 +1,20 @@
+use re_types::{components::Material, datatypes::Rgba32};
+use re_viewer_context::ViewerContext;
+
+use crate::color::edit_color_ui;
+
+// TODO(andreas): as we add more elements to the Material struct, we'll need a multi-line editor.
+//                  begs the question though if a material struct is really the right way to go!
+pub fn edit_material_ui(
```

Comment:
Related (not this PR): https://github.com/rerun-io/rerun/issues/6624#issuecomment-2191158915